crates/dc_bundle/src/definition/view.rs

```diff
@@ -13,5 +13,115 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+use crate::definition::element::line_height::LineHeightType;
+use crate::definition::element::num_or_var::NumOrVarType;
+use crate::definition::element::{
+    background, Background, Color, ColorOrVar, FontFeature, FontStretch, FontStyle, FontWeight,
+    Hyperlink, LineHeight, NumOrVar, TextDecoration,
+};
 
 include!(concat!(env!("OUT_DIR"), "/designcompose.definition.view.rs"));
+
+impl TextStyle {
+    fn new_default() -> Self {
+        TextStyle {
+            text_color: Some(Background::new(background::BackgroundType::Solid(
+                ColorOrVar::new_color(Color::BLACK),
+            ))),
+            font_size: Some(NumOrVar { num_or_var_type: Some(NumOrVarType::Num(18.0)) }),
+            font_family: None,
+            font_weight: Some(FontWeight::NORMAL),
+            font_style: FontStyle::Normal.into(),
+            font_stretch: Some(FontStretch::NORMAL),
+            letter_spacing: 0.0,
+            text_decoration: TextDecoration::None.into(),
+            line_height: Some(LineHeight { line_height_type: Some(LineHeightType::Percent(1.0)) }),
+            font_features: Vec::new(),
+            hyperlink: None,
+        }
+    }
+}
+
+impl StyledTextRun {
+    pub fn new(label: impl ToString) -> StyledTextRun {
+        StyledTextRun { text: label.to_string(), style: Some(TextStyle::new_default()) }
+    }
+    pub fn bold(self) -> Self {
+        StyledTextRun {
+            style: Some(TextStyle {
+                font_weight: Some(FontWeight::BOLD),
+                ..self.style.expect("This was just created")
```

Comment:
Is this `.expect()` because `style` is now an `Option<>`? What does "This was just created" mean?

Reply:
Yes, and it's because the structure was created within the same function, so it's very unlikely to have changed. Is there a cleaner way to handle this?

Reply:
I don't think the structure was created within the same function. You could so something like:
```
let test = StyledTextRun::new("hi");
let test_bold = test.bold();
```
Which is fine. If you didn't use the `new()` function and did something like:
```
let test = StyledTextRun { text: "hi".to_string(), style: None };
let test_bold = test.bold();
```
Then it would fail with the `expect()`. But no, I don't know a better way to do this.

Reply:
It looks like you eliminated this, but I think the normal way would be to wrap the actual type and not an Option (so the outer type models the state), or create a default style when it's None and define the `Default` trait impl for the inner type.

Reply:
Yeah, it looks like nothing's actually using the functions, so I'm just deleting them.